flowmachine/tests/test_exposed_datetime_mixin.py

```diff
@@ -0,0 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from flowmachine.core.mixins import ExposedDatetimeMixin
+from datetime import datetime
+
+
+class DummyDatetime(ExposedDatetimeMixin):
+    def __init__(self, start_date, end_date):
+        self.start_date = start_date
+        self.end_date = end_date
+
+
+def test_exposed_datetime_mixin():
```

Comment:
Might be good to parametrise this test, to test that passing `datetime.date` or `datetime.datetime` arguments has the same outcome.